apps/fluent-tester/src/TestComponents/CheckboxV1/CheckboxV1Test.tsx

```diff
@@ -21,7 +21,7 @@ function onChangeUncontrolled(_e: InteractionEvent, isChecked: boolean) {
 const BasicCheckbox: React.FunctionComponent = () => {
   return (
     <View>
-      <Checkbox label="Unchecked checkbox (undefined)" onChange={onChangeUncontrolled} />
+      <Checkbox label="Unchecked checkbox (undefined) aidasdijk ahsdihsaidhas jasdhasid" onChange={onChangeUncontrolled} />
```

Comment:
I think we should leave the previous text only. 
As we have already done our testing, this is not required for tester.